source/getting-started/user-guide/amds-interface/index.md

```diff
@@ -209,29 +220,43 @@ The AMDS-AMDC digital interface sometimes needs debugging if valid data does not
 
-The value of `V` should be non-zero if the connection is active.
-Each time you read the counters, the value of `V` should increment.
+If all the counters are stuck at zero, the mostly likely cause is that the AMDS is not enabled in the timing manager. If only the `Timed out data` counter is incrementing, the AMDS is either unplugged, or the GPIO mux configuration was not completed correctly. If the connection is made correctly and active, the value of the `Valid` counter should be continually incrementing.
 
-Note that, since the UART link has two data lines, a separate 16-bit counter is used for each data line.
+```{note}
+For all counters, since the UART link has two data lines, a separate 16-bit counter is used for each data line.
 The `counters` command returns both counters concatenated into a 32-bit value.
 For example, if the value is `0x0002` for the first UART data line and `0x0003` for the second UART data line, the counter will appear as: `0x00020003`.
+```
 
 #### Example
 
 Consider running this experiment: the AMDS-AMDC link is working and you are reading valid data. Then, the cable comes unplugged.
 
-- When the link is working, the `V` counter will continuously be incrementing (and wrapping around at the 16-bit limit)
-- As the cable comes unplugged, chances are, it will occur during some byte transmission. This will cause a corrupt data byte and the `C` counter will increment a few times based on exactly how the cable became unplugged.
-- After the cable is fully unplugged, the AMDC views every byte requested as a timeout, so the `T` counter will continuously increment. The other counters should remain frozen in their last value.
+- When the link is working, the `Valid` counter will continuously be incrementing (and wrapping around at the 16-bit limit)
+- As the cable comes unplugged, chances are, it will occur during some byte transmission. This will cause some  corrupt data bytes and some individual bytes to timeout. The `Corrupt` and `Timed out bytes` counters may each increment a few times based on exactly how the cable became unplugged.
+- After the cable is fully unplugged, the AMDC views every data stream requested as a data timeout, so the `Timed out data` counter will continuously increment. The other counters should remain frozen in their last value.
+
+This debug counter behaviour can be seen occuring repeatedly in the following video captured during experimental testing. When connected, `Valid` is incrementing, and when disconnected, `Timed out data` is incrementing. The `Corrupt` and `Timed out bytes` counters increment as the transition is made between the connected and disconnected states.
+
+![](images/debug-counters.gif)
+
+The image below also shows a graphical capture of the counters incrementing their values.
+
+```{image} images/debug-counter-graph.png
+:alt: Graph showing the values of debug counters during experiment
+:height: 18em
+:align: left
+```
```

Comment:
Nice! Can you give the reader some help as to what the traces on the plot are?

Either replot with better legend entries (might be too hard due to requiring lab time), or, add a table here which explains the legend entries and gives a couple sentences to explain the graph.

Reply:
done